widgetssdk/src/main/java/com/glia/widgets/helper/ResourceProvider.java

```diff
@@ -78,6 +80,14 @@ public int convertDpToIntPixel(float dp) {
         return Math.round(convertDpToPixel(dp));
     }
 
+    @Override
+    public String getResourceKey(int stringKey) {
+        /*
+            getResourceName wil contain package and resourceName separated by "/", discarding the first package part
```

Comment:
I think several examples might be useful here. At least for me it's a bit difficult to imagine how it will actually look like because I'm not that deed in the project context.